etc/base.yaml

```diff
@@ -295,23 +297,59 @@ timescaledb:
       # Uncomment when upgrading
       #existingClaim: "data-timescaledb-postgresql-0"
 
+# Make sure to set:
```

Comment:
We can, but this would it would require a different approach of configuration that would touch dependencies of other services as well. I consider this a separate feature which is out of scope for this PR. I have filed a new Issue to for lose this idea.

Reply:
We decided to leave the current method in place.